frontend/src/scenes/pipeline/PipelineNode.tsx

```diff
@@ -46,30 +48,33 @@ export const scene: SceneExport = {
 export function PipelineNode(params: { stage?: string; id?: string } = {}): JSX.Element {
     const { stage, id } = paramsToProps({ params })
 
-    const { currentTab } = useValues(pipelineNodeLogic)
+    const { currentTab, node } = useValues(pipelineNodeLogic)
 
     if (!stage) {
         return <NotFound object="pipeline stage" />
     }
 
-    const tabToContent: Record<PipelineNodeTab, JSX.Element> = {
+    const tabToContent: Partial<Record<PipelineNodeTab, JSX.Element>> = {
         [PipelineNodeTab.Configuration]: <PipelineNodeConfiguration />,
         [PipelineNodeTab.Metrics]: <PipelineNodeMetrics id={id} />,
         [PipelineNodeTab.Logs]: <PipelineNodeLogs id={id} stage={stage} />,
         [PipelineNodeTab.History]: <ActivityLog id={id} scope={ActivityScope.PLUGIN} />,
     }
+    if (node.backend === PipelineBackend.BatchExport) {
```

Comment:
Being picky but History should really be the last tab and Runs should be before it